### DISCUSSION TITLE
目录问题

### DISCUSSION THREAD

**Discussion by slzhoufdu2024** [stone-zeng/fduthesis#324] [Q&A] [answered]
请问大家，目录中如何不显示插图目录和符号表？

**Comment by stone-zeng** [comment, marked as answer]
导言区加上：

```tex
\makeatletter
\ExplSyntaxOn
\__fdu_patch_cmd:Nnn \listoffigures
  { \__fdu_chapter:V }
  { \__fdu_chapter_no_toc:V }
\__fdu_patch_cmd:Nnn \__fdu_notation_begin:
  { \__fdu_chapter:V }
  { \__fdu_chapter_no_toc:V }
\ExplSyntaxOff
\makeatother
```

**Comment by slzhoufdu2024** [comment]
问题已解决，感谢学长！！
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Fri, Mar 15, 2024 09:35 PM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [stone-zeng/fduthesis] 目录问题 (Discussion #324)

&nbsp;



 
导言区加上：
 \makeatletter \ExplSyntaxOn \__fdu_patch_cmd:Nnn \listoffigures   { \__fdu_chapter:V }   { \__fdu_chapter_no_toc:V } \__fdu_patch_cmd:Nnn \__fdu_notation_begin:   { \__fdu_chapter:V }   { \__fdu_chapter_no_toc:V } \ExplSyntaxOff \makeatother
